ubuntu-packages.txt

```diff
@@ -4,7 +4,6 @@ git
 graphviz
 inkscape
 latexmk
-plantuml
```

Comment:
I think we need to have `plantuml` here. It is required for the python binding.

Reply:
I put plantuml back to how it was originally